README.md

```diff
@@ -42,7 +42,7 @@ The most common ones are the following:
 
 ```
 RESTY_IMAGE_BASE=ubuntu|centos|rhel|debian|alpine|amazonlinux
-RESTY_IMAGE_TAG=xenial|bionic|focal|6|7|8|stretch|buster|latest|latest
+RESTY_IMAGE_TAG=18.04|20.04|6|7|8|9|10|11|latest|latest
```

Comment:
why not 16.04 as well?

```suggestion
RESTY_IMAGE_TAG=16.04|18.04|20.04|6|7|8|9|10|11|latest|latest
```

Reply:
mainly because it's old and eol.

Reply:
Ubuntu 16.04 has only reached end of standard support not EOL; that doesn't occur until [April 2026](https://wiki.ubuntu.com/Releases).